.travis.yml

```diff
@@ -67,7 +67,8 @@ addons:
       - libopenmpi-dev
       - libgsl0-dev
       - libfftw3-dev
-      - python-pip
+      - python3-pip
+      - clang-format
```

Comment:
Does this need to specifically be `clang-format-10`?

Reply:
We could do that, usually the clang never gets bump within a ubuntu release, so that should be ok for a while.